code/game/gamemodes/clock_cult/clock_mobs.dm

```diff
@@ -213,66 +234,138 @@
 			var/resulthealth
 			resulthealth = round((abs(config.health_threshold_dead - host.health) / abs(config.health_threshold_dead - host.maxHealth)) * 100)
 			stat(null, "Host Health: [resulthealth]%")
-			if(resulthealth > 60)
-				stat(null, "You are [recovering ? "unable to deploy" : "can deploy on hearing True Name"]!")
+			if(ratvar_awakens)
+				stat(null, "You are able to deploy!")
 			else
-				stat(null, "You are [recovering ? "unable to deploy" : "can deploy to protect your host"]!")
-		stat(null, "You do [melee_damage_upper] on melee attacks.")
+				if(resulthealth > 60)
+					stat(null, "You are [recovering ? "unable to deploy" : "can deploy on hearing your True Name"]!")
+				else
+					stat(null, "You are [recovering ? "unable to deploy" : "can deploy to protect your host"]!")
+		stat(null, "You do [melee_damage_upper] damage on melee attacks.")
+
+/mob/living/simple_animal/hostile/clockwork/marauder/adjustHealth(amount) //Fatigue damage
+	var/fatiguedamage = adjust_fatigue(amount)
+	if(amount > 0)
+		for(var/mob/living/L in view(2, src))
+			if(istype(L.l_hand, /obj/item/weapon/nullrod) || istype(L.r_hand, /obj/item/weapon/nullrod)) //Null rods allow direct damage
+				src << "<span class='userdanger'>The presence of a brandished holy artifact weakens your armor!</span>"
+				amount *= 5 //if a wielded null rod is nearby, it takes five times the health damage
+				break
+	return ..() + fatiguedamage
```

Comment:
I think you may need to explicitly include the arguments, as the variable "amount" might change during the proc, before parent proc is called.